vector/src/main/res/values/strings.xml

```diff
@@ -2946,10 +2965,6 @@
     <string name="error_failed_to_import_keys">Failed to import keys</string>
 
     <string name="settings_notification_configuration">Notifications configuration</string>
-    <string name="settings_messages_at_room">Messages containing @room</string>
-    <string name="settings_messages_in_e2e_one_to_one">Encrypted messages in one-to-one chats</string>
-    <string name="settings_messages_in_e2e_group_chat">Encrypted messages in group chats</string>
-    <string name="settings_when_rooms_are_upgraded">When rooms are upgraded</string>
```

Comment:
Lint will complain that we have translation but not original string. So either remove all the existing translations or keep the original string and we do the cleanup in the strings from time to time.
I would go for the first option.

Reply:
My bad, actually those strings has just been moved above. Please ignore my comment